elasticapm/contrib/django/client.py

```diff
@@ -36,6 +36,10 @@
 from django.core.exceptions import DisallowedHost
 from django.db import DatabaseError
 from django.http import HttpRequest
+try:
+    from rest_framework.request import Request
```

Comment:
could we rename this to `DrfRequest` to make it a bit clearer against what kind of request class we're making an instance check?

```suggestion
    from rest_framework.request import Request as DrfRequest